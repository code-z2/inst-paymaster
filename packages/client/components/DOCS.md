# Paymaster

This is a description of the components , stylesheets and
pages that makes up paymaster project.

## Components

This project makes use of tailwindcss and css modules.

Each component is styled using a local stylesheet. This makes it easy to identify where to make changes for any section or component in the page.

The navbar/footer component and other global variables are styled using a global stylesheet.

Components bear similar names to it's stylesheet modules
for easy identification.

E.g `landingpage/firstSection` styling can be seen in `styles/landing/firstsection.module.css`

Adopted Css grid method [Multi-grid-one-page-layout](https://medium.com/@nikkipantony/multi-grid-one-page-layout-css-grid-6efefd537404);

The components in this directory are:

## Active paymaster

- **activeNav**: This is the navbar of the active paymaster's page.
  `activepaymaster/activeNav`
  - styling: open `styles/navbar.css` to view the navbar styling.
    **All the navbars in the project are styled from**
    `styles/navbar.css`

```
 .active-paymaster-nav {
    @apply paynav;
    }

.active-paymaster-nav .list-ul {
@apply col-start-[-1] w-full grid grid-cols-[1fr_2fr] px-1 content-center text-xs sm:text-sm gap-10 font-light sm:font-normal;
}

```

    The`active-paymaster-nav ` class is reusing the same styling with the navbar in the landing page .

To view the paymasters page navigate to `http://localhost:3000/auth/{auth_id}/activepaymaster`

- **allpaymasters** : This is the paymasters list component.
  `activepaymaster/allpaymasters` styling for this component and the media queries is found at `styles/activepaymaster/allpaymasters.module.css`

- **firstSection** : It is the first part i.e first section of the active paymasters page

- **paymastersListPage** : This is where all the components in the active paymaster directory are assembled together and then exported to the active paymasters route.
  `activepaymaster/paymastersListPage` is exported to `/auth/[private]/activepaymaster`

## Auth directory

- **private**: A HOC for authentication using NEXT_PUBLIC_AUTH_ID environmental variable

## landing page

- **firstSection** : This is the hero section of the landing page.

- **landing** : This is where all the components in the landing page directory are assembled together and then exported to the landing page route.
  `activepaymaster/landingpage` is exported to `/auth/[private]`

**Applied padding for windows scaling b/w 100 to 125**

```

/*************scaling 100 to 125***********************/

@media (min-width: 1100px) and (-webkit-min-device-pixel-ratio: 1.25) {
  .scaleto100 {
    @apply px-20;
  }
}


```

`tailwind.config.css` custom configurations

```

 theme: {
    extend: {
      colors: {
        backgroundTheme: "#11142b",
        secondaryGlow: "#7d0cc1",
        jsClientGlow: "#7D0CC1",
        jsClientDark: "rgba(125, 12, 193, 0.3)",
        blurColor: "rgba(125, 12, 193, 0.5)",
        flowerBlur: "rgba(193, 12, 12, 0.5)",
      },
      fontFamily: {
        monserrat: "Montserrat",
        color: "#fff",
      },
      shadow: {
        btn: "4px 4px 14px rgba(0, 0, 0, 0.15)",
      },


```
