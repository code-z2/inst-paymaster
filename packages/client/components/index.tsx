// Reusable exports
export { default as Image } from "next/image";
export { default as Link } from "next/link";

// Landing page components
export { default as LandingPage } from "./landingpage/Default";

export { default as LandingPageLayout } from "./layouts/Default";
export { UnSecureLandingPage } from "./landingpage/Default";
export { default as Hero } from "./landingpage/Hero";
export { default as RebatesMiddleware } from "./landingpage/RebatesMiddleware";
export { default as BuildPaymasters } from "./landingpage/BuildPaymasters";

// authentication HOC component
export { default as Auth } from "./auth/private";

// paymasters list page components
export { default as PaymastersList } from "./activepaymaster/paymastersListPage";
export { default as Loadmore } from "./activepaymaster/loadmore";
export { default as ActiveNav } from "./activepaymaster/activeNav";
export { default as FirstSection } from "./activepaymaster/firstSection";
export { default as Allpaymasters } from "./activepaymaster/allpaymasters";
