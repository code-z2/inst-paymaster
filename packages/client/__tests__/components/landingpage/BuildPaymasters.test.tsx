import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import { BuildPaymasters } from "../../../components";

/*
 *  snapshots of the buildpaymasters section components
 */
describe("Build paymasters Section ", () => {
  it("should render the build components ", () => {
    const home = render(<BuildPaymasters />);
    expect(home).toMatchSnapshot();
  });
});
