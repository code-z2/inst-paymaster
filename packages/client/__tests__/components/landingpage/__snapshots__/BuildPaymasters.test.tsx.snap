// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Build paymasters Section  should render the build components  1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="section-group"
      >
        <section
          class="build-section site-section"
        >
          <div
            class="wrapper"
          >
            <header
              class="section-header py-32 w-image inverse"
            >
              <div
                class="section-content"
              >
                <h2
                  class="font-semibold text-5xl"
                >
                  Build paymasters
                </h2>
                <p>
                  Integrate our typescript SDK to form paymasters compatible zk transaction objects in your dapps.
                </p>
                <a
                  href="/dev"
                >
                  <button
                    class="cta"
                  >
                    Explore SDK
                  </button>
                </a>
              </div>
              <div
                class="section-img-cont"
              >
                <img
                  alt="paymaster codeblock"
                  class="p-16"
                  data-nimg="1"
                  decoding="async"
                  height="300"
                  loading="lazy"
                  src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
                  srcset="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
                  style="color: transparent;"
                  width="300"
                />
              </div>
            </header>
            <ul
              class="options"
            >
              <li
                class="option"
              >
                <article
                  class="option-item"
                >
                  <h3
                    class="font-semibold text-xl"
                  >
                    JS Client Library
                  </h3>
                  <p>
                    Lorem ipsum dolor sit amet consectetur. Sit lectus ac et turpis eu ultricies odio praesent in.
                  </p>
                  <button
                    class="cta none"
                  >
                    <img
                      alt="npm icon"
                      class="icon !w-8 !h-8"
                      data-nimg="1"
                      decoding="async"
                      height="32"
                      loading="lazy"
                      src="/_next/image?url=%2Fimg.jpg&w=64&q=75"
                      srcset="/_next/image?url=%2Fimg.jpg&w=32&q=75 1x, /_next/image?url=%2Fimg.jpg&w=64&q=75 2x"
                      style="color: transparent;"
                      width="32"
                    />
                    <span
                      class="text text-paymasters-purple"
                    >
                      JS Docs
                    </span>
                  </button>
                </article>
              </li>
              <li
                class="option"
              >
                <article
                  class="option-item"
                >
                  <h3
                    class="font-semibold text-xl"
                  >
                    HTTP API
                  </h3>
                  <p>
                    Lorem ipsum dolor sit amet consectetur. Sit lectus ac et turpis eu ultricies odio praesent in.
                  </p>
                  <button
                    class="cta none"
                  >
                    <img
                      alt="npm icon"
                      class="icon !w-8 !h-8"
                      data-nimg="1"
                      decoding="async"
                      height="32"
                      loading="lazy"
                      src="/_next/image?url=%2Fimg.jpg&w=64&q=75"
                      srcset="/_next/image?url=%2Fimg.jpg&w=32&q=75 1x, /_next/image?url=%2Fimg.jpg&w=64&q=75 2x"
                      style="color: transparent;"
                      width="32"
                    />
                    <span
                      class="text text-paymasters-purple"
                    >
                      HTTP Docs
                    </span>
                  </button>
                </article>
              </li>
              <li
                class="option"
              >
                <article
                  class="option-item"
                >
                  <h3
                    class="font-semibold text-xl"
                  >
                    Web App
                  </h3>
                  <p>
                    Lorem ipsum dolor sit amet consectetur. Sit lectus ac et turpis eu ultricies odio praesent in.
                  </p>
                  <button
                    class="cta none"
                  >
                    <img
                      alt="npm icon"
                      class="icon !w-8 !h-8"
                      data-nimg="1"
                      decoding="async"
                      height="32"
                      loading="lazy"
                      src="/_next/image?url=%2Fimg.jpg&w=64&q=75"
                      srcset="/_next/image?url=%2Fimg.jpg&w=32&q=75 1x, /_next/image?url=%2Fimg.jpg&w=64&q=75 2x"
                      style="color: transparent;"
                      width="32"
                    />
                    <span
                      class="text text-paymasters-purple"
                    >
                      Make your first upload
                    </span>
                  </button>
                </article>
              </li>
            </ul>
            <section
              class="supported-languages"
            >
              <h3
                class="font-semibold text-xl"
              >
                Supported Languages
              </h3>
              <ul
                class="supported-languages-list"
              >
                <li
                  class="item"
                >
                  <button
                    class="cta"
                  >
                    Nodejs
                  </button>
                </li>
                <li
                  class="item"
                >
                  <button
                    class="cta"
                  >
                    Solidity
                  </button>
                </li>
                <li
                  class="item"
                >
                  <button
                    class="cta"
                  >
                    JavaScript
                  </button>
                </li>
                <li
                  class="item"
                >
                  <button
                    class="cta"
                  >
                    TypeScript
                  </button>
                </li>
                <li
                  class="item"
                >
                  <button
                    class="cta"
                  >
                    Cairo
                  </button>
                </li>
              </ul>
            </section>
          </div>
        </section>
      </div>
      <section
        class="site-section partners-section"
      >
        <div
          class="wrapper"
        >
          <header
            class="section-header"
          >
            <h2
              class="font-semibold text-5xl text-center"
            >
              Our Partners
            </h2>
          </header>
          <ul
            class="partners-list"
          >
            <li
              class="partner-item"
            >
              <div
                class="img-cont flex items-center justify-center w-12 h-w-12"
              >
                <img
                  alt="zkSynk"
                  data-nimg="1"
                  decoding="async"
                  height="64"
                  loading="lazy"
                  src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                  srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                  style="color: transparent;"
                  width="64"
                />
              </div>
              <h3
                class="font-medium text-lg"
              >
                zkSynk
              </h3>
            </li>
            <li
              class="partner-item"
            >
              <div
                class="img-cont flex items-center justify-center w-12 h-w-12"
              >
                <img
                  alt="StrakNet"
                  data-nimg="1"
                  decoding="async"
                  height="64"
                  loading="lazy"
                  src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                  srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                  style="color: transparent;"
                  width="64"
                />
              </div>
              <h3
                class="font-medium text-lg"
              >
                StrakNet
              </h3>
            </li>
            <li
              class="partner-item"
            >
              <div
                class="img-cont flex items-center justify-center w-12 h-w-12"
              >
                <img
                  alt="Braavos"
                  data-nimg="1"
                  decoding="async"
                  height="64"
                  loading="lazy"
                  src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                  srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                  style="color: transparent;"
                  width="64"
                />
              </div>
              <h3
                class="font-medium text-lg"
              >
                Braavos
              </h3>
            </li>
            <li
              class="partner-item"
            >
              <div
                class="img-cont flex items-center justify-center w-12 h-w-12"
              >
                <img
                  alt="Argrent"
                  data-nimg="1"
                  decoding="async"
                  height="64"
                  loading="lazy"
                  src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                  srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                  style="color: transparent;"
                  width="64"
                />
              </div>
              <h3
                class="font-medium text-lg"
              >
                Argrent
              </h3>
            </li>
            <li
              class="partner-item"
            >
              <div
                class="img-cont flex items-center justify-center w-12 h-w-12"
              >
                <img
                  alt="Chainlink"
                  data-nimg="1"
                  decoding="async"
                  height="64"
                  loading="lazy"
                  src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                  srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                  style="color: transparent;"
                  width="64"
                />
              </div>
              <h3
                class="font-medium text-lg"
              >
                Chainlink
              </h3>
            </li>
            <li
              class="partner-item"
            >
              <div
                class="img-cont flex items-center justify-center w-12 h-w-12"
              >
                <img
                  alt="Outlier VC"
                  data-nimg="1"
                  decoding="async"
                  height="64"
                  loading="lazy"
                  src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                  srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                  style="color: transparent;"
                  width="64"
                />
              </div>
              <h3
                class="font-medium text-lg"
              >
                Outlier VC
              </h3>
            </li>
            <li
              class="partner-item"
            >
              <div
                class="img-cont flex items-center justify-center w-12 h-w-12"
              >
                <img
                  alt="Developer Dao"
                  data-nimg="1"
                  decoding="async"
                  height="64"
                  loading="lazy"
                  src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                  srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                  style="color: transparent;"
                  width="64"
                />
              </div>
              <h3
                class="font-medium text-lg"
              >
                Developer Dao
              </h3>
            </li>
          </ul>
        </div>
      </section>
      <section
        class="cta-section site-section"
      >
        <div
          class="wrapper"
        >
          <header
            class="section-header py-32 w-image inverse"
          >
            <div
              class="section-content"
            >
              <h2
                class="font-semibold text-4xl"
              >
                Get started, explore and create Paymasters
              </h2>
              <a
                href="/dev"
              >
                <button
                  class="cta dark"
                >
                  Launch App
                </button>
              </a>
            </div>
            <div
              class="section-img-cont"
            >
              <img
                alt="paymaster logo"
                class="p-16"
                data-nimg="1"
                decoding="async"
                height="300"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
                style="color: transparent;"
                width="300"
              />
            </div>
          </header>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <div
      class="section-group"
    >
      <section
        class="build-section site-section"
      >
        <div
          class="wrapper"
        >
          <header
            class="section-header py-32 w-image inverse"
          >
            <div
              class="section-content"
            >
              <h2
                class="font-semibold text-5xl"
              >
                Build paymasters
              </h2>
              <p>
                Integrate our typescript SDK to form paymasters compatible zk transaction objects in your dapps.
              </p>
              <a
                href="/dev"
              >
                <button
                  class="cta"
                >
                  Explore SDK
                </button>
              </a>
            </div>
            <div
              class="section-img-cont"
            >
              <img
                alt="paymaster codeblock"
                class="p-16"
                data-nimg="1"
                decoding="async"
                height="300"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
                style="color: transparent;"
                width="300"
              />
            </div>
          </header>
          <ul
            class="options"
          >
            <li
              class="option"
            >
              <article
                class="option-item"
              >
                <h3
                  class="font-semibold text-xl"
                >
                  JS Client Library
                </h3>
                <p>
                  Lorem ipsum dolor sit amet consectetur. Sit lectus ac et turpis eu ultricies odio praesent in.
                </p>
                <button
                  class="cta none"
                >
                  <img
                    alt="npm icon"
                    class="icon !w-8 !h-8"
                    data-nimg="1"
                    decoding="async"
                    height="32"
                    loading="lazy"
                    src="/_next/image?url=%2Fimg.jpg&w=64&q=75"
                    srcset="/_next/image?url=%2Fimg.jpg&w=32&q=75 1x, /_next/image?url=%2Fimg.jpg&w=64&q=75 2x"
                    style="color: transparent;"
                    width="32"
                  />
                  <span
                    class="text text-paymasters-purple"
                  >
                    JS Docs
                  </span>
                </button>
              </article>
            </li>
            <li
              class="option"
            >
              <article
                class="option-item"
              >
                <h3
                  class="font-semibold text-xl"
                >
                  HTTP API
                </h3>
                <p>
                  Lorem ipsum dolor sit amet consectetur. Sit lectus ac et turpis eu ultricies odio praesent in.
                </p>
                <button
                  class="cta none"
                >
                  <img
                    alt="npm icon"
                    class="icon !w-8 !h-8"
                    data-nimg="1"
                    decoding="async"
                    height="32"
                    loading="lazy"
                    src="/_next/image?url=%2Fimg.jpg&w=64&q=75"
                    srcset="/_next/image?url=%2Fimg.jpg&w=32&q=75 1x, /_next/image?url=%2Fimg.jpg&w=64&q=75 2x"
                    style="color: transparent;"
                    width="32"
                  />
                  <span
                    class="text text-paymasters-purple"
                  >
                    HTTP Docs
                  </span>
                </button>
              </article>
            </li>
            <li
              class="option"
            >
              <article
                class="option-item"
              >
                <h3
                  class="font-semibold text-xl"
                >
                  Web App
                </h3>
                <p>
                  Lorem ipsum dolor sit amet consectetur. Sit lectus ac et turpis eu ultricies odio praesent in.
                </p>
                <button
                  class="cta none"
                >
                  <img
                    alt="npm icon"
                    class="icon !w-8 !h-8"
                    data-nimg="1"
                    decoding="async"
                    height="32"
                    loading="lazy"
                    src="/_next/image?url=%2Fimg.jpg&w=64&q=75"
                    srcset="/_next/image?url=%2Fimg.jpg&w=32&q=75 1x, /_next/image?url=%2Fimg.jpg&w=64&q=75 2x"
                    style="color: transparent;"
                    width="32"
                  />
                  <span
                    class="text text-paymasters-purple"
                  >
                    Make your first upload
                  </span>
                </button>
              </article>
            </li>
          </ul>
          <section
            class="supported-languages"
          >
            <h3
              class="font-semibold text-xl"
            >
              Supported Languages
            </h3>
            <ul
              class="supported-languages-list"
            >
              <li
                class="item"
              >
                <button
                  class="cta"
                >
                  Nodejs
                </button>
              </li>
              <li
                class="item"
              >
                <button
                  class="cta"
                >
                  Solidity
                </button>
              </li>
              <li
                class="item"
              >
                <button
                  class="cta"
                >
                  JavaScript
                </button>
              </li>
              <li
                class="item"
              >
                <button
                  class="cta"
                >
                  TypeScript
                </button>
              </li>
              <li
                class="item"
              >
                <button
                  class="cta"
                >
                  Cairo
                </button>
              </li>
            </ul>
          </section>
        </div>
      </section>
    </div>
    <section
      class="site-section partners-section"
    >
      <div
        class="wrapper"
      >
        <header
          class="section-header"
        >
          <h2
            class="font-semibold text-5xl text-center"
          >
            Our Partners
          </h2>
        </header>
        <ul
          class="partners-list"
        >
          <li
            class="partner-item"
          >
            <div
              class="img-cont flex items-center justify-center w-12 h-w-12"
            >
              <img
                alt="zkSynk"
                data-nimg="1"
                decoding="async"
                height="64"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                style="color: transparent;"
                width="64"
              />
            </div>
            <h3
              class="font-medium text-lg"
            >
              zkSynk
            </h3>
          </li>
          <li
            class="partner-item"
          >
            <div
              class="img-cont flex items-center justify-center w-12 h-w-12"
            >
              <img
                alt="StrakNet"
                data-nimg="1"
                decoding="async"
                height="64"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                style="color: transparent;"
                width="64"
              />
            </div>
            <h3
              class="font-medium text-lg"
            >
              StrakNet
            </h3>
          </li>
          <li
            class="partner-item"
          >
            <div
              class="img-cont flex items-center justify-center w-12 h-w-12"
            >
              <img
                alt="Braavos"
                data-nimg="1"
                decoding="async"
                height="64"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                style="color: transparent;"
                width="64"
              />
            </div>
            <h3
              class="font-medium text-lg"
            >
              Braavos
            </h3>
          </li>
          <li
            class="partner-item"
          >
            <div
              class="img-cont flex items-center justify-center w-12 h-w-12"
            >
              <img
                alt="Argrent"
                data-nimg="1"
                decoding="async"
                height="64"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                style="color: transparent;"
                width="64"
              />
            </div>
            <h3
              class="font-medium text-lg"
            >
              Argrent
            </h3>
          </li>
          <li
            class="partner-item"
          >
            <div
              class="img-cont flex items-center justify-center w-12 h-w-12"
            >
              <img
                alt="Chainlink"
                data-nimg="1"
                decoding="async"
                height="64"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                style="color: transparent;"
                width="64"
              />
            </div>
            <h3
              class="font-medium text-lg"
            >
              Chainlink
            </h3>
          </li>
          <li
            class="partner-item"
          >
            <div
              class="img-cont flex items-center justify-center w-12 h-w-12"
            >
              <img
                alt="Outlier VC"
                data-nimg="1"
                decoding="async"
                height="64"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                style="color: transparent;"
                width="64"
              />
            </div>
            <h3
              class="font-medium text-lg"
            >
              Outlier VC
            </h3>
          </li>
          <li
            class="partner-item"
          >
            <div
              class="img-cont flex items-center justify-center w-12 h-w-12"
            >
              <img
                alt="Developer Dao"
                data-nimg="1"
                decoding="async"
                height="64"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=128&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=64&q=75 1x, /_next/image?url=%2Fimg.jpg&w=128&q=75 2x"
                style="color: transparent;"
                width="64"
              />
            </div>
            <h3
              class="font-medium text-lg"
            >
              Developer Dao
            </h3>
          </li>
        </ul>
      </div>
    </section>
    <section
      class="cta-section site-section"
    >
      <div
        class="wrapper"
      >
        <header
          class="section-header py-32 w-image inverse"
        >
          <div
            class="section-content"
          >
            <h2
              class="font-semibold text-4xl"
            >
              Get started, explore and create Paymasters
            </h2>
            <a
              href="/dev"
            >
              <button
                class="cta dark"
              >
                Launch App
              </button>
            </a>
          </div>
          <div
            class="section-img-cont"
          >
            <img
              alt="paymaster logo"
              class="p-16"
              data-nimg="1"
              decoding="async"
              height="300"
              loading="lazy"
              src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
              srcset="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
              style="color: transparent;"
              width="300"
            />
          </div>
        </header>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
